ray_on_golem/server/services/golem/golem.py

```diff
@@ -296,12 +307,9 @@ async def _upload_node_configuration(
         context: WorkContext,
         ip: str,
         ssh_public_key_data: str,
-        *,
-        add_state_log: Callable[[str], Awaitable[None]],
     ):
-        provider_desc = await self._get_provider_desc(context)
+        provider_desc = await self.get_provider_desc(context.activity)
         logger.info(f"Running initial commands on {provider_desc}, {ip=}, {context.activity=}")
-        await add_state_log("[6/9] Running bootstrap commands...")
```

Comment:
Any reason for moving state logging outside of functions? It was aimed to keep consistency across all steps.

Reply:
Yes, so (almost)all are called from a single method now and it is easier to find them. A top of that it was needed to perform the little refactor around `_verify_ssh_connection_check`